src/mapml/control/ScaleBar.js

```diff
@@ -0,0 +1,48 @@
+export var ScaleBar = L.Control.extend({
+    options: {
+      metric: true,
```

Comment:
I think you should use the value specified by `M.options.whatever` to restrict the scale to metric or imperial, but not both.